keras/losses/losses.py

```diff
@@ -1858,3 +1858,41 @@ def binary_focal_crossentropy(
         focal_bce = weight * focal_bce
 
     return ops.mean(focal_bce, axis=axis)
+
+
+@keras_export(
+    [
+        "keras.metrics.npairs_loss",
+        "keras.losses.npairs_loss",
+    ]
+)
+def npairs_loss(y_true, y_pred):
```

Comment:
Let's not add it to the API -- keep the implementation in the example code.

Reply:
done